src/detect-detection-filter.c

```diff
@@ -611,24 +579,16 @@ static int DetectDetectionFilterTestSig3(void)
     drops += ((PacketTestAction(p, ACTION_DROP)) ? 1 : 0);
     p->action = 0;
 
-    if (alerts == 3 && drops == 3)
```

Comment:
what is there to replace these checks?

I think what needs to be done is:
1. remove the alerts and drops counters
2. use FAIL* macros for all `PacketAlertCheck` calls

Reply:
One thing I am confused about is:
According to the above the test passes if `if (alerts == 3 && drops == 3)`
If I remove counters with FAIl/PASS macros then I will be failing or passing on all of these rather then a specific alert or drop count. Is that okay?
The alert and drop variables are getting incremented(or not) 7 times in the above lines.